artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerTarget.java

```diff
@@ -95,6 +97,22 @@ public static MirrorController getControllerInUse() {
       return CONTROLLER_THREAD_LOCAL.get();
    }
 
+   /** The rate in milliseconds that we will print OperationContext debug information on the mirror target */
+   private static final int DEBUG_CONTEXT_RATE;
+   private ScheduledFuture scheduledRateDebug = null;
+
+   static {
+      int rate;
+      try {
+         rate = Integer.parseInt(System.getProperty(AMQPMirrorControllerTarget.class.getName() + ".DEBUG_CONTEXT_RATE", "5000"));
+      } catch (Throwable e) {
+         logger.debug(e.getMessage(), e);
+         rate  = 0;
+      }
+
+      DEBUG_CONTEXT_RATE = rate;
```

Comment:
As this is more setting the period / interval rather than rate, perhaps DEBUG_CONTEXT_PERIOD ?